conf/css/postcss.config.js

```diff
@@ -16,8 +16,22 @@ module.exports = {
       filter: path => ! RegExp('^/static').test(path)
     }),
     require('postcss-extend')(),
+    require('postcss-preset-env')({
+      // SEE: https://github.com/csstools/postcss-preset-env#features
+      stage: false,
+      // SEE: https://github.com/csstools/postcss-preset-env/blob/master/src/lib/plugins-by-id.js#L35
+      features: {
+        'custom-media-queries': true,
+        'media-query-ranges': true,
+      }
+    }),
     require('cssnano')({
-      preset: 'default'
+      preset: ['default', {
+        // Disabled to avoid sweeping bad CSS under the rug
+        // NOTE: Also fixes comparability in tests by retaining expectation code
+        discardDuplicates: { exclude: true },
+        mergeRules: { exclude: true },
+      }]
```

Comment:
1. Do __not__ remove duplicate CSS. _I want to see that, so I can't solve the problem at the source._
2. Do __not__ merge selectors that have the same rulesets. _This is actually a nice feature, but it's ruining my ability to test `postcss-preset-env`…_